.zenodo.json

```
@@ -20,6 +20,11 @@
         "name": "Andriyash, Igor",
         "orcid": "0000-0003-0313-4496"
       },
+      {
+        "affiliation": "Lawrence Livermore National Laboratory",
+        "name": "Angus, Justin R.",
```

Like this? @JustinRayAngus 
```suggestion
        "name": "Angus, Justin Ray",
```

---

Either way is fine with me. Orcid looks correct.